pbj-core/pbj-compiler/src/main/java/com/hedera/pbj/compiler/impl/Common.java

```diff
@@ -254,6 +254,8 @@ public static String getFieldsHashCode(final List<Field> fields, String generate
                                 result = 31 * result + Integer.hashCode($fieldName.protoOrdinal());
                              }
                              """).replace("$fieldName", f.nameCamelFirstLower());
+				} else if (f.type() == Field.FieldType.MAP) {
+					generatedCodeSoFar += getMapHashCodeGeneration(generatedCodeSoFar, f);
```

Comment:
This is a part of the maps implementation that has been in mainline for a long time already. I suggest to remerge/rebase your branch so that the PR only shows your actual changes.